### PR TITLE
Feat/form creation

### PR DESCRIPTION
**Pull request by MihlaliMgwebi**
Create and Delete Trip

### CODE REVIEW

File: src/app/components/trip-create/trip-create.component.html [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
-    </div>
-    <div class="form-group">
-      <label for="itineraryDescription">Itinerary Description</label>
-      <textarea class="form-control" id="itineraryDescription" formControlName="description"></textarea>
-    </div>
-  </div>
-  <button type="submit" class="btn btn-primary">Create Trip</button>
-
+<form nz-form [formGroup]="tripForm" (ngSubmit)="submitForm()">
+  <!--  Title  -->
```
Comment (kphutiyagae) [inline src/app/components/trip-create/trip-create.component.html:2]: Remove comment perhaps?

**Review by kphutiyagae — APPROVED** [review]
Looks okay.

**Comment by kphutiyagae** [comment]
Please amend comments before merge.

File: src/app/components/trip-create/trip-create.component.html (hunk 1 above)
Comment (MihlaliMgwebi) [reply src/app/components/trip-create/trip-create.component.html:2]: I keep them to read my html better. Thank you for the suggestion

**Pull request merged**